### PR TITLE
Adding Chat Completions API Samples & modifying folder structure.

### PR DESCRIPTION
**Pull request by RiyaBaliyan123**

### CODE REVIEW

**Review by shshubhe — APPROVED** [review]

File: Basic_Samples/Chat/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Refer to the following documents for a better understanding of Azure OpenAI conc
  - [How to generate or manipulate text](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/completions) 
  - [What are embeddings?](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/concepts/understand-embeddings)
  - [How to generate embeddings?](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/embeddings?tabs=console)
+ - [How to work with the ChatGPT?](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/chatgpt?pivots=programming-language-chat-completions)
```
Comment (dereklegenzoff) [inline Basic_Samples/Chat/README.md:22]: Could we remove the question mark?
```suggestion
 - [How to work with the ChatGPT](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/chatgpt?pivots=programming-language-chat-completions)
```

File: Basic_Samples/Completions/README.md [hunk 2]
```diff
@@ -61,12 +62,15 @@ One can get the OPENAI_API_KEY value from the Azure Portal. Go to https://portal
          export OPENAI_API_KEY="REPLACE_WITH_YOUR_KEY_VALUE_HERE"
 
 - For _Completions_ scenario, one can start with using your model name ("COMPLETIONS_MODEL" in _config.json_ file) as "text_davinci_003". <br>
+And for Chat GPT Completions, one can start with using your model name as "gpt-35-turbo" ("CHAT_GPT_MODEL" in _config.json_ file).
```
Comment (dereklegenzoff) [inline Basic_Samples/Completions/README.md:65]: ChatGPT is one word

File: Basic_Samples/Chat/config.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "CHAT_GPT_MODEL":"<ChatGPT Model Name>",
```
Comment (dereklegenzoff) [inline Basic_Samples/Chat/config.json:2]: Could we change this to "CHATGPT_MODEL"? In general i'd like to make ChatGPT a single word across the repo

**Review by dereklegenzoff — COMMENTED** [review]
Looks good. Left a few minor comments. I'd also vote to get rid of the top level `Basic_Samples` folder and make the folders inside of it all top level folders.

**Comment by shshubhe** [comment]
> Looks good. Left a few minor comments. I'd also vote to get rid of the top level `Basic_Samples` folder and make the folders inside of it all top level folders.

@dereklegenzoff We want to maintain this folder structure. In the future, we might get more basic samples (DALL-E, whisper?). Good to have a consolidated folder structure from the beginning itself.

**Review by shshubhe — APPROVED** [review]

**Pull request merged**